src/main/java/com/glovo/ttglovo/securityManagement/appuser/AppUser.java

```diff
@@ -0,0 +1,172 @@
+package com.glovo.ttglovo.securityManagement.appuser;
+
+import com.glovo.ttglovo.cart.CartItem;
+import lombok.*;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+import java.util.*;
+
+@Builder
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity(name = "AppUser")
+@Table(name = "user_table",
+        uniqueConstraints = {
+                @UniqueConstraint(
+                        columnNames = "email",
+                        name = "uk_email"
+                )
+        }
+)
+public class AppUser implements UserDetails {
+
+    @Id
+    @SequenceGenerator(
+            name = "user_sequence",
+            sequenceName = "user_sequence",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "user_sequence"
+    )
+    @Column(name = "id")
+    private Long id;
+
+    @NotEmpty(message = "First name may not be empty")
+    @Column(
+            name = "first_name",
```

Comment:
The name generated for the db table is usually derived correctly from the attribute name. Was it different than _first_name_ in this case?